src/main/java/racingcar/Winner.java

```diff
@@ -0,0 +1,29 @@
+package racingcar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winner {
+
+    private int findMaxPosition(List<Car> cars) {
+        return cars.stream()
+                .mapToInt(Car::getPosition)
+                .max()
+                .orElse(-1);
+    }
+
+    public String findWinner(List<Car> cars) {
+        int maxPosition = findMaxPosition(cars);
+
+        List<String> winnerList = cars.stream()
+                .filter(car -> car.getPosition() == maxPosition)
+                .map(car -> car.getName())
+                .collect(Collectors.toList());
+
+        String winner = winnerList.stream()
+                .collect(Collectors.joining(", "));
+
+        return winner;
```

Comment:
아래와 같이 수정해 볼 수 있을 것 같아요. 😄 

```suggestion
        return String.join(", ", winnerList);
```